src/Domain/Entity/Group.php

```diff
@@ -4,5 +4,13 @@
 
 abstract class Group extends CoreEntity
 {
+    public function isTlec(): bool
+    {
+        return true;
+    }
 
+    public function isTleo(): bool
+    {
+        return true;
```

Comment:
Probably should just return false. In fact, a group cannot be the parent of a programme so it's utterly academic anyway.